models/tasks.js

```diff
@@ -67,13 +67,13 @@ const fetchTasks = async () => {
 
 const fetchActiveTaskMembers = async () => {
   try {
-    const tasksSnapshot = await tasksModel.where('status', '==', 'active').get()
+    const tasksSnapshot = await tasksModel.where('type', '==', 'feature').where('status', '==', 'active').get()
     const activeMembers = []
     tasksSnapshot.forEach((task) => {
       const taskData = task.data()
-      if (taskData.participants) {
+      if (taskData.assignee) {
         activeMembers.push(
-          ...taskData.participants
+          taskData.assignee
```

Comment:
This will then return those members for which assigned task status is `pending` 🤔 
Is it expected?

Reply:
This wont be returning assignees with assigned status pending as there is extra condition to check for status active

Reply:
That's what I'm saying 😅
An user can have only 1 active task
So let's see they forget to change the status of one of their pending tasks to active, they'll be shown idle, right?

Or if there's someone completed a task and if they have some task in pending and they forget to mark it as active, they'll be shown idle, right?

Reply:
Yeah I had a similar discussion, the way forward we need to make sure what ever the task we are working currently we need to mark it as active to keep the active track of all the progress ahead and assign to others accordingly if there are any issues

Reply:
Yeah you are right , they will be shown idle and some one would be asking them in the group leading them to mark it active

Reply:
The user can mark their tasks as `active` from http://my.realdevsquad.com/tasks 👍🏻